### PR TITLE
Add manage.py method to update operators documentation. 

### PR DESCRIPTION
**Pull request by cs2be**
Algorithm will always update documentation for "develop" branch and will conditionally update operator docs for version != '0.9.0' or '0.10.0' (Will not update operator if site.operators.en.json exists).

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -21,8 +21,10 @@ before_install:
   - export PATH=$PATH:$HOME/.local/bin # put aws in the path
   - chmod +x scripts/travis/build-docker.sh scripts/travis/deploy-image.sh
 script:
-  - ./scripts/travis/build-docker.sh
-  - ./scripts/travis/deploy-image.sh
+  - |
+    if [ "$TRAVIS_EVENT_TYPE" == "cron" ]; then python portal/manage.py update_operator_docs; exit 0; fi;
```
Comment (varunarora) [inline .travis.yml]: Didnt know you could run manage.py from anywhere. That's cool!

File: portal/deploy/operators.py [hunk 2]
```diff
@@ -88,8 +95,6 @@ def clean_json_string(body):
 
     except ValueError, e:
         if str(e).startswith('Invalid control character'):
-            print 'Operators JSON reading/loading failed because: ', e
-
             faulty_character_index = int(re.search(
```
Comment (varunarora) [inline portal/deploy/operators.py]: Hadn't you explicitly asked me to print a logging message for this?

File: portal/deploy/sitemap_generator.py [hunk 3]
```diff
@@ -120,23 +120,23 @@ def _create_sphinx_site_map(parent_list, node, language):
...
-            })
+            sections = section['sections']
+            break
+
+    if sections:
+        sections.append({
+            '$ref': {
+                lang: 'documentation/%s' % get_operator_sitemap_name(lang)
+            }
+        })
```
Comment (varunarora) [inline portal/deploy/sitemap_generator.py]: I read the command file and so, good use of the pattern!

File: portal/portal/management/commands/update_operator_docs.py [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+                    full_path = '%s/%s/documentation' % (path, dir)
+
+                    if dir == 'develop':
+                        # always update operator docs for 'develop' branch
+                        self._update_operator_docs_for_version(full_path, dir)
+                    elif version_re.match(dir):
+                        if dir != '0.9.0' and dir != '0.10.0':
+                            # Only add operators for version != 0.9.0 or 0.10.0
+                            existing_sitemap_path = '%s/%s' % (full_path, get_operator_sitemap_name('en'))
+
```
Comment (varunarora) [inline portal/portal/management/commands/update_operator_docs.py]: Interesting

File: portal/portal/management/commands/update_operator_docs.py [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+            if version == 'develop':
+                docker_image_name = DOCKER_IMAGE_NAME_TEMPLATE % 'latest'
+            else:
+                docker_image_name = DOCKER_IMAGE_NAME_TEMPLATE % version
+
+            print 'Pulling docker image: %s' % docker_image_name
+            client.images.pull(docker_image_name)
+
+            print 'Generating operator docs...'
+            operator_doc_json_string = client.containers.run(docker_image_name, 'print_operators_doc')
```
Comment (varunarora) [inline portal/portal/management/commands/update_operator_docs.py]: Had no idea you could do this

File: portal/portal/management/commands/update_operator_docs.py [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+            output_path = get_destination_documentation_dir(version, 'documentation')
```
Comment (varunarora) [inline portal/portal/management/commands/update_operator_docs.py]: 👍

File: portal/deploy/operators.py (hunk 2 above)
Comment (cs2be) [reply portal/deploy/operators.py]: Did I? I don't remember :(...

Comment (varunarora) [reply portal/deploy/operators.py]: Yep pretty sure 😄

**Pull request merged**